src/pages/task/logs/logTypes/LogMessageLine.tsx

```diff
@@ -2,6 +2,9 @@ import React from "react";
 import { LogMessage } from "gql/generated/types";
 import { getLogLineWrapper } from "pages/task/logs/logTypes/logMessageLine/LogLines";
 import { format } from "date-fns";
+import linkifyHtml from "linkifyjs/html";
+var Convert = require("ansi-to-html");
```

Comment:
Is it not possible to import this? If not be sure to use let

Reply:
oops!!